src/helpers/CamelJBang.ts

```diff
@@ -123,6 +126,42 @@ export class CamelJBang {
 		return new ShellExecution(this.jbang, [...this.defaultJbangArgs, 'cmd', `${operation}-route`, integration, `--id=${routeId}`]);
 	}
 
+	public async getRuntimeInfoFromMavenContext(integrationFilePath: string): Promise<string | undefined> {
```

Comment:
currently providing a string which is the full json string returned by Camel JBang. Potentially possible to provide a specific object. It will be more Type-safed and easier to manipulate. In this case, not sure where this Type should be defined.

Reply:
Yes, you're right, Kaoto should define the type, and then the host application should fulfil it.